ci-operator/step-registry/cucushift/chainupgrade/toimage/cucushift-chainupgrade-toimage-commands.sh

```diff
@@ -998,6 +1053,13 @@ for target in "${TARGET_RELEASES[@]}"; do
     fi
     check_history
     health_check
+    currentVersion=$(oc version -o yaml | grep openshiftVersion | grep -o '[0-9]*[.][0-9]*' | head -1)
```

Comment:
maybe we can use global variable TARGET_VERSION or TARGET_MINOR_VERSION, because after health_check currentVersion should be same with TARGET_VERSION otherwise this step should fail.

Reply:
thanks @JianLi-RH 
hmm, I'm not sure the TARGET_VERSION or TARGET_MINOR_VERSION if will be changed, eg. 4.17 or 4.18.
So I get the current cluster version which should be more accurate I think.

Reply:
agree.

Reply:
Ok, updated. thanks. @JianLi-RH @jianlinliu